bugbug/db.py

```diff
@@ -37,11 +37,11 @@ def is_registered(path: str) -> bool:
     return path in DATABASES
 
 
-def exists(path):
+def exists(path) -> bool:
```

Comment:
Type annotations should be added for function arguments too

Reply:
ack, have added that in the latest commit